aspnetcore/diagnostics/asp0015.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "ASP0015: Suggest using IHeaderDictionary properties"
+description: "Learn about analysis rule ASP0015: Suggest using IHeaderDictionary properties"
+author: tdykstra
+monikerRange: '>= aspnetcore-7.0'
```

Comment:
All of these are actually going to be going into 8.0

Reply:
> All of these are actually going to be going into 8.0

Why wouldn't they be in all versions `'>= aspnetcore-7.0'`?

If this content changes in future versions, we'll handle it then.

Reply:
I changed moniker range for these articles to `>= aspnetcore-8.0`

Reply:
Changing to 8.0 moniker range resulted in build error:

>Invalid moniker range: '>= aspnetcore-8.0'. Moniker 'aspnetcore-8.0' is not defined.

So I'm changing back to 7.0. When we get the moniker created we'll update these articles.